src/Conference/ConferenceDetails.test.js

```diff
@@ -33,26 +34,23 @@ describe("ConferenceDetails", () => {
     }
   };
 
-  let wrapper;
-  const findElement = identifier => {
-    return wrapper.find(identifier).get(0).props.children;
-  };
-
   it("renders a logged-in conference details page", async () => {
     expect.assertions(8);
 
     const history = createMemoryHistory();
     history.push("/1");
 
     act(() => {
-      wrapper = mount(
-        <Router history={history}>
-          <ConferenceDetails
-            conference={mockData}
-            id="1"
-            allCookies={mockCookie}
-          />
-        </Router>
+      setWrapper(
```

Comment:
Could go one further and setMountedWrapper, then mount the components in the util